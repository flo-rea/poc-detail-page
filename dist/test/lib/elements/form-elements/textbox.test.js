"use strict";
const React = require('react');
const renderer = require('react/lib/ReactTestRenderer');
const textbox_1 = require('../../../../src/lib/view/elements/form-elements/textbox');
jest.mock('react-dom');
describe('Textbox', () => {
    const component = renderer.create(React.createElement(textbox_1.TextBox, {id: "textbox1", name: "item2"}, "This is a sample textbox."));
    it('has correct default options', () => {
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});
//# sourceMappingURL=textbox.test.js.map