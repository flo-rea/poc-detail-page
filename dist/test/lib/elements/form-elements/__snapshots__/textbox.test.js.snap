exports[`Textbox has correct default options 1`] = `
<textarea
  id="textbox1"
  name="item2">
  This is a sample textbox.
</textarea>
`;
